java/src/jmri/jmrix/cmri/serial/networkdriver/ConnectionConfig.java

```diff
@@ -38,7 +38,9 @@ public String name() {
 
     public void loadDetails(JPanel details) {
 
-        b.addActionListener(new NodeConfigAction((CMRISystemConnectionMemo)adapter.getSystemConnectionMemo()));
+        if (adapter != null) {
+            b.addActionListener(new NodeConfigAction((CMRISystemConnectionMemo)adapter.getSystemConnectionMemo()));
```

Comment:
If this runs with adapter==null, it won't crash right away, but it will behave oddly later on (because the listener won't be present).  I'm much rather see a Large Amount of information being logged here so we have some chance of tracking down the real problem, which is the null adapter value.